### PR TITLE
Disable or enable exchange access

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by stephamon** [comment]
@cyrichardson can you take a look at this one? I applied your feedback from the similar article for Rackspace Email, so there might not be anything else for this one! Thanks!

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a9f3703501/how-to/).

**Comment by cyrichardson** [comment]
@stephamon Looking now.

File: content/exchange/disable-or-enable-exchange-email-access.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+### Prerequisites
+
+- **Applies to:** Administrator
+- **Difficulty:** Easy
+- **Time needed:** Approximately 10 minutes
+- **Tools required:** Cloud Office Control Panel access
+
+For more information about prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+Blocking access to a mailbox prevents users from logging in to that mailbox. You might need to block access to a mailbox if an employee has left your company, or if you suspect the account has been compromised.
```
Comment (catlook) [inline content/exchange/disable-or-enable-exchange-email-access.md]: ... company or ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-14af8d0214/how-to/).

**Pull request merged**